src/main/resources/wa-task-initiation-civil-civil.dmn

```diff
@@ -585,6 +585,59 @@ else
   false</text>
         </inputExpression>
       </input>
+      <input id="InputClause_0j2n0e1" label="Discontinuance Task Type" camunda:inputVariable="discontinuanceTaskType">
+        <inputExpression id="LiteralExpression_0vfy6y7" typeRef="string">
+          <text>if(additionalData !=null 
+and additionalData.Data != null
+and additionalData.Data.courtPermissionNeeded!= null
+and additionalData.Data.courtPermissionNeeded="YES") 
+then 
+"VERIFY_COURT_PERMISSION"
+else if(additionalData != null
+and additionalData.Data != null
+and additionalData.Data.selectedClaimantForDiscontinuance!= null
+and additionalData.Data.selectedClaimantForDiscontinuance!="Both" 
+and additionalData.Data.courtPermissionNeeded!= null
+and additionalData.Data.courtPermissionNeeded="NO"
+and additionalData.Data.typeOfDiscontinuance!=null
+and additionalData.Data.typeOfDiscontinuance="FULL_DISCONTINUANCE")
+then 
+"TAKE_OFFLINE"
+else if(additionalData != null
+and additionalData.Data != null 
+and additionalData.Data.isDiscontinuingAgainstBothDefendants!= null
+and additionalData.Data.isDiscontinuingAgainstBothDefendants="NO"
+and additionalData.Data.courtPermissionNeeded!= null
+and additionalData.Data.courtPermissionNeeded="NO"
+and additionalData.Data.typeOfDiscontinuance!=null
+and additionalData.Data.typeOfDiscontinuance ="FULL_DISCONTINUANCE")
+then 
+"TAKE_OFFLINE"
+else if(additionalData != null
+and additionalData.Data != null
+and ((additionalData.Data.selectedClaimantForDiscontinuance != null
+and additionalData.Data.selectedClaimantForDiscontinuance="Both") or
+(additionalData.Data.selectedClaimantForDiscontinuance = null)) 
+and additionalData.Data.courtPermissionNeeded!= null
+and additionalData.Data.courtPermissionNeeded="NO"
+and additionalData.Data.typeOfDiscontinuance!=null
+and additionalData.Data.typeOfDiscontinuance ="FULL_DISCONTINUANCE") 
+then 
+"REMOVE_HEARNING"
+else if (additionalData != null and additionalData.Data != null 
+and ((additionalData.Data.isDiscontinuingAgainstBothDefendants!= null
+and additionalData.Data.isDiscontinuingAgainstBothDefendants="NO") or
```

Comment:
This condition is previously fulfilled:
(additionalData.Data.isDiscontinuingAgainstBothDefendants!= null
and additionalData.Data.isDiscontinuingAgainstBothDefendants="NO")